upgrade_test.py

```diff
@@ -501,6 +504,11 @@ def shuffle_nodes_and_alternate_dcs(nodes: List[BaseNode]) -> List[BaseNode]:
 
         return [x for x in chain.from_iterable(zip_longest(*dc_nodes.values())) if x]
 
+    @staticmethod
+    def _update_scylla_yaml_on_node(node_to_update: BaseNode, updates: dict):
+        with ScyllaYamlUpdateEvent(node=node_to_update.name, update=updates), node_to_update.remote_scylla_yaml() as scylla_yaml:
```

Comment:
maybe add this to `remote_scylla_yaml()` to always send event on yaml update?
This is quite important action, so would be worth doing it always?
there's a diff extracted in `sdcm.cluster_k8s.KubernetesCluster.manage_file_in_scylla_config_map` and `sdcm.cluster.BaseNode.remote_scylla_yaml` (maybe some other place would need update, need to check) - so this diff could be published as event.
@KnifeyMoloko @fruch @fgelcer what do you think?

Reply:
Seems reasonable to me.